pkg/source/source_test.go

```diff
@@ -226,6 +226,27 @@ var _ = Describe("Source", func() {
 			Expect(err.Error()).To(Equal("cache did not sync"))
 
 		})
```

Comment:
Looks like we have 0 (also before this PR) tests for source.Informer.

If it's relatively simple can we have those start/stop tests for the informer source as well?